change/@internal-react-composites-d8da4cf2-00e1-48b9-84d0-0e7dfcd0f453.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
In this case it doesn't matter too much (since we're going to merge back into release/1.2.0).

This would be a 'minor' change in my opinion even though there is no API change because it is a large UX change.